superset/config.py

```diff
@@ -1335,6 +1335,7 @@ def EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument
 MENU_HIDE_USER_INFO = False
 
 # Set to False to only allow viewing own recent activity
+# or to disallow users from viewing other users profile page
```

Comment:
Looking at the code we are just removing the links to the profiles. Is there anything else that is actually blocking the user to visit a profile?

Reply:
Yes, on the backend change, the call to `get_user_activity_access_error` will assert if `ENABLE_BROAD_ACTIVITY_ACCESS` is `True` or not, if it's `False` and the user_id is not equal to the current user, the response will be 403 with an error msg. Following same pattern already in-place on other endpoint on `/superset`.

Reply:
Ok thanks for adding that